extensions/amp-auto-ads/amp-auto-ads.md

```diff
@@ -155,6 +171,24 @@ fetched from a 3rd party ad network at the URL defined in
       position.
     </td>
   </tr>
+  <tr>
+    <td>attributes</td>
+    <td>No</td>
+    <td>Object&lt;string, string&gt;</td>
+    <td>{}</td>
+    <td>
+      A map from attribute name to value for attrbiutes to apply to all
+      <code>&lt;amp-ad&gt;</code>s injected using this placement. An attribute
```

Comment:
Instead of `<code>&lt;amp-ad&gt;</code>s` how about `<code>&lt;amp-ad&gt;</code> elements`